playground/ExternalServices/.aspire/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "appHostPath": "../ExternalServices.AppHost/ExternalServices.AppHost.csproj"
```

Comment:
Should these be checked in? @mitchdenny 

If not, can I more strongly suggest that `.aspire` be added to the git ignore.

Reply:
Shouldn't be checked in. Feel free to add it to .gitignore.